lib/abstract_notifier/testing/minitest.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Minitest
```

Comment:
We should put this under AbstractNotifier namespace. Let's use `AbstractNotifier::TestHelper`.

Reply:
@palkan Thank you for your suggestion!

Should I rename the file to `lib/abstract_notifier/test_helper` ?